app/bundles/CampaignBundle/Entity/Event.php

```diff
@@ -246,6 +254,12 @@ public static function loadMetadata(ORM\ClassMetadata $metadata)
             ->columnName('channel_id')
             ->nullable()
             ->build();
+        /** CAPTIVEA.CORE START **/
+        $builder->createManyToOne('scoringCategory', 'Mautic\ScoringBundle\Entity\ScoringCategory')
+            ->inversedBy('usedByEvents')
+            ->addJoinColumn('scoringcategory_id', 'id', true, false, 'SET NULL')
+            ->build();
```

Comment:
For each database schema change there has to be a migration. Generate a new one with `app/console mautic:migrations:generate` It will generate a new class in app/migrations folder. Fill in the content. You can inspire in the previous migrations.